### DISCUSSION TITLE
请将MCBBS永久关站的消息加入新闻主页

### DISCUSSION THREAD

**Discussion by heerxingen** [Light-Beacon/PCL2-NewsHomepage#109] [Ideas]
如题，不pull request是因为我不会写PCL主页（呜）

**Comment by Light-Beacon** [comment]
其实最上边提示栏已经写了
1a8ab147fc8a4c91505199b3817a1734d260fd72

**Comment by heerxingen** [comment]
那个是暂时关站，我说的是永久关站并且不再重启的通知

**Comment by Light-Beacon** [comment]
照搬公告了：
![image](https://github.com/Light-Beacon/PCL2-NewsHomepage/assets/49828867/2ca48882-fd58-4af7-a89d-dc86bdcfeb47)
